.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Use Docker in rootless mode (To solve "Permission denied" error when removing test_app directory)
+        uses: ScribeMD/rootless-docker@0.2.2
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - shell: bash
+        run: |
+          ./bootstrap.sh
```

Comment:
Currently we built DUNE docker image from scratch (which tests bootstrap.sh and Dockerfile).
I prefer building from scratch than just downloading latest DUNE image from packages (because Dockerfile and bootstrap.sh might change in the meanwhile).